tests/script-based-pre/cargo_autoharness_contracts/contracts.expected

```diff
@@ -37,23 +41,37 @@ arithmetic_overflow\
 	 - Status: SUCCESS\
 	 - Description: "attempt to compute `unchecked_mul` which would overflow"
 
+Autoharness: Checking function should_pass::alignment::Alignment::as_usize's contract against all possible inputs...
+
+should_pass::alignment::Alignment::as_usize\
+	 - Status: SUCCESS\
+	 - Description: "Rust intrinsic assumption failed"
```

Comment:
The tests suggest that the "Rust intrinsic" part of that description doesn't exist.